### ISSUE TITLE
13beta1 and beta3  Connot access external storege (local USB HDD)

### ISSUE THREAD

**Issue by janumix** [nextcloud/server#7437]
### Steps to reproduce
1. Set up local HDD in External storage (or try to use already configured before upgrade to 13b1)
2. Try click on Files->(ExtHDD) - nothing happens.

### Expected behaviour
Just access ExtHDD 

### Actual behaviour
When you click on ExtHDD nothing happens

### Server configuration

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)
13 beta 1
13 beta 3

**Updated from an older Nextcloud/ownCloud or fresh install:**
Updated from 12.0.4

**Where did you install Nextcloud from:**
via web installer

**Signing status:**

No errors have been found.

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
Enabled:
  - activity: 2.6.1
  - calendar: 1.5.7
  - comments: 1.3.0
  - contacts: 2.0.1
  - dav: 1.4.5
  - federatedfilesharing: 1.3.1
  - files: 1.8.0
  - files_external: 1.4.1
  - files_pdfviewer: 1.2.0
  - files_sharing: 1.5.0
  - files_texteditor: 2.5.1
  - files_trashbin: 1.3.0
  - files_versions: 1.6.0
  - files_videoplayer: 1.2.0
  - firstrunwizard: 2.2.1
  - gallery: 18.0.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.1.0
  - news: 11.0.5
  - nextcloud_announcements: 1.2.0
  - notifications: 2.1.0
  - oauth2: 1.1.0
  - password_policy: 1.3.0
  - previewgenerator: 1.0.7
  - provisioning_api: 1.3.0
  - serverinfo: 1.3.0
  - sharebymail: 1.3.0
  - survey_client: 1.1.0
  - theming: 1.4.1
  - twofactor_backupcodes: 1.2.3
  - updatenotification: 1.3.0
  - workflowengine: 1.3.0
Disabled:
  - admin_audit
  - bruteforcesettings
  - encryption
  - federation
  - files_clipboard
  - keeweb
  - systemtags
  - user_external
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

instanceid’ => ‘xxx’,
‘passwordsalt’ => ‘xxx’,
‘secret’ => ‘xxx’,
‘memcache.local’ => ‘\OC\Memcache\APCu’,
‘trusted_domains’ =>
array (
0 => ‘jxxxxt’,
1 => ‘192.168.100.28’,
2 => ‘10.95.0.186’,
),
‘datadirectory’ => ‘/config/www/nextcloud/data’,
‘overwrite.cli.url’ => ‘https://xxx’,
‘dbtype’ => ‘mysql’,
‘version’ => ‘13.0.0.6’,
‘dbname’ => ‘nextcloud’,
‘dbhost’ => ‘172.17.0.2:3306’,
‘dbport’ => ‘’,
‘dbtableprefix’ => ‘oc_’,
‘dbuser’ => ‘xxxx’,
‘dbpassword’ => ‘xxxx’,
‘logtimezone’ => ‘UTC’,
‘installed’ => true,
‘mail_smtpmode’ => ‘smtp’,
‘mail_smtpsecure’ => ‘tls’,
‘mail_from_address’ => ‘xxxxx’,
‘mail_domain’ => ‘gmail.com’,
‘mail_smtpauthtype’ => ‘LOGIN’,
‘mail_smtpauth’ => 1,
‘mail_smtphost’ => ‘smtp.gmail.com’,
‘mail_smtpport’ => ‘587’,
‘mail_smtpname’ => ‘xxx’,
‘mail_smtppassword’ => ‘xxx’,
‘maintenance’ => false,
‘theme’ => ‘’,
‘loglevel’ => 3,
‘updater.release.channel’ => ‘beta’,
‘mysql.utf8mb4’ => true,
‘preview_max_x’ => 1024,
‘preview_max_y’ => 1024,
‘preview_max_scale_factor’ => 1,

</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...
local: /var/media/hdd

**Are you using encryption:** yes/no
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
Webdav

### Client configuration
Chrome 54.0.2840.71

**Operating system:**
Linux Ubuntu 17.10
Linux Ubuntu 17.04

#### Web server error log
<details>
<summary>Web server error log</summary>
nothing here related to external storage 
The only entry refers to non-existing googlestorage  but this is different issue
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by janumix** [comment]
Folder icon is not presented as previously (with arrow inside).
After clicking on its name gray filed remains but forlder didn't open.

![image](https://user-images.githubusercontent.com/952615/33983692-603cf214-e0b5-11e7-8ba4-436b6615696e.png)

**Comment by janumix** [comment]
here is the same behariour in "External storage" - the only difference is icon.

![image](https://user-images.githubusercontent.com/952615/33983954-3f21cf4a-e0b6-11e7-8e02-5db34124321b.png)

**Comment by ziqingshi** [comment]
I have the same problem.You should click the share button first,then you can access external_storage.

**Comment by janumix** [comment]
You're right. It works - thanks. Sharing external storage should be enabled. 
Nevertheless issue need to be open until fix will be made.

**Comment by MorrisJobke** [comment]
Fixed by #7658
